.github/actions/core-cicd/prepare-runner/README.md

```diff
@@ -53,7 +53,7 @@ Here's an example of how to incorporate this action into your workflow:
 ```yaml
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Suggest updating this to reflect how you've implemented with fallback, or even with no fallback at all

```suggestion
    runs-on: ubuntu-${{ vars.UBUNTU_RUNNER_VERSION || '22.04' }}
```